sbi/inference/posteriors/ensemble_posterior.py

```diff
@@ -16,7 +16,7 @@
 from sbi.utils.user_input_checks import process_x
 
 
-class EnsemblePosterior(NeuralPosterior):
+class EnsemblePosterior(NeuralPotentialPosterior):
```

Comment:
In principle we should be able to make ensemble score-based posteriors as well. Might be worth doing before release.